ksql-common/src/main/java/io/confluent/ksql/configdef/UnmodifiableConfigDef.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.configdef;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import org.apache.kafka.common.config.ConfigDef;
+
+/**
+ * Wrapper around ConfigDef to disable modification.
+ */
+public final class UnmodifiableConfigDef extends ConfigDef {
```

Comment:
`ConfigDef` is mutable. This is just a quick and dirty wrapper to make it immutable. Main reason for this is that I didn't feel comfortable adding `ConfigDef` to the list of known immutable types in the model tests that test for immutable types.